### DISCUSSION TITLE
全局变量保存与读取

### DISCUSSION THREAD

**Discussion by fanfanSH** [ctripcorp/flybirds#17] [Feature Request]
web ui测试中，经常遇到url拼接参数，如果参数是动态的（如产品id），或者参数容易失效（如token或session），此时需要自动获取到对应参数并保存到全局变量中，供url拼接或其他场景使用。

**Comment by clgwlg** [comment]
v0.3.4 版本，增加了相关支持
https://github.com/ctripcorp/flybirds/releases/tag/v0.3.4
